egs2/voxtlm/lm1/local/prepare_bpe_text.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-i", "--inp", type=str, help="Input file with all training data"
+    )
+    parser.add_argument("-o", "--out", type=str, help="Output file for bpe training")
```

Comment:
```suggestion
    parser.add_argument("-o", "--output", type=str, help="Output file for bpe training")
```
ditto
We usually use more complete words or phrases for `--` based options